radarr.json

```diff
@@ -4,6 +4,7 @@
     "artifact": "https://github.com/freenas/iocage-plugin-radarr.git",
     "official": false,
     "properties": {
+        "allow_mlock": "1",
```

Comment:
Actually can we please have this as an integer `1` without the quotes ?

Reply:
sure! done!